memory/libhugetlbfs.py

```diff
@@ -108,7 +107,7 @@ def setUp(self):
                             % (pages_available, pages_requested))
 
         git.get_repo('https://github.com/libhugetlbfs/libhugetlbfs.git',
-                     destination_dir=self.workdir)
+                     branch='next', destination_dir=self.workdir)
```

Comment:
@harish-24 so next branch have all the patches which able to run test suite in python 3 ?

Reply:
@PraveenPenguin yes, that is right. Once it is merged into master (expected to be least 3 months) we can move to master back.

Reply:
@harish-24 sure , will try once and merge